snapshot/lib/snapshot/runner.rb

```diff
@@ -204,6 +207,21 @@ def erase_simulator(device_type)
       `xcrun simctl erase #{device_udid} &> /dev/null`
     end
 
+    def localize_simulator(device_type, language, locale)
+      apple_locale = language.sub("-", "_")
```

Comment:
Do we or Apple support 3-part locale tags (e.g. [`zh_Hant_HK`](http://www.localeplanet.com/icu/zh-Hant-HK/index.html))? If so, might it be better to `gsub` here to change all dashes to underscores?